test/doc/core_1_refresher.cpp

```diff
@@ -288,7 +288,7 @@ struct associated_allocator<handler, Allocator>
 template<class Executor>
 struct associated_executor<handler, Executor>
 {
-    using type = boost::asio::executor;
+    using type = boost::asio::any_io_executor;
```

Comment:
`net::`

Reply:
Especially important as this appears in the docs

Reply:
net:: won't compile here